[2024-11-03T17:45:31.651+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T17:45:31.656+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.json_to_csv_review_data manual__2024-11-03T17:44:50.172889+00:00 [queued]>
[2024-11-03T17:45:31.659+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.json_to_csv_review_data manual__2024-11-03T17:44:50.172889+00:00 [queued]>
[2024-11-03T17:45:31.659+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 3
[2024-11-03T17:45:31.663+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): json_to_csv_review_data> on 2024-11-03 17:44:50.172889+00:00
[2024-11-03T17:45:31.665+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=113) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T17:45:31.666+0000] {standard_task_runner.py:72} INFO - Started process 115 to run task
[2024-11-03T17:45:31.667+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'json_to_csv_review_data', 'manual__2024-11-03T17:44:50.172889+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpq9e0565q']
[2024-11-03T17:45:31.668+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask json_to_csv_review_data
[2024-11-03T17:45:31.683+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.json_to_csv_review_data manual__2024-11-03T17:44:50.172889+00:00 [running]> on host 9ab0960d13ae
[2024-11-03T17:45:31.705+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='json_to_csv_review_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T17:44:50.172889+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-03T17:44:50.172889+00:00'
[2024-11-03T17:45:31.705+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T17:45:39.650+0000] {python.py:240} INFO - Done. Returned value was: Successfully converted Data/Raw/TEST/test_user_reviews.json to CSV and uploaded to Data/Raw_CSV/TEST/test_user_reviews.csv
[2024-11-03T17:45:39.663+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-03T17:45:39.664+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=json_to_csv_review_data, run_id=manual__2024-11-03T17:44:50.172889+00:00, execution_date=20241103T174450, start_date=20241103T174531, end_date=20241103T174539
[2024-11-03T17:45:39.695+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-03T17:45:39.706+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-03T17:45:39.707+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
